resources/tracing/templates/kyma-additions/zipkin-service.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     app.kubernetes.io/component: zipkin 
 spec:
   ports:
-  - name: jaeger-collector-zipkin
+  - name: tcp-jaeger-collector-zipkin
```

Comment:
We figured out that zipkin is also a http based protocol. Please change this prefix as well.